indexing/src/main/resources/org/allenai/common/indexing/indexing.conf

```diff
@@ -1,136 +1,88 @@
-org.allenai.common.indexing: {
+include "mappingHelpers.conf"
 
-  stoplist: {
+org.allenai.common.indexing {
+  stoplist {
     group: "org.allenai.nlp.resources"
     name: "smart_stopwords_2.txt"
     version: 1
   }
-
 }
 
-org.allenai.common.indexing.base: {
-  elasticSearch: {
+org.allenai.common.indexing.base {
+  elasticSearch {
     // Specify the right clustername. It usually defaults to `elasticsearch` but
     // you can verify by using the command:
     // curl -XGET 'http://localhost:9200/_cluster/health?pretty=true'.
-    // For Aristo Solvers, the hostIp should be changed to `aristo-es1.dev.ai2`
+    // For Aristo Solvers, the hostIp should be changed to `aristo-es.dev.ai2`
     // with clusterName `aristo-es`.
     clusterName: "elasticsearch"
     hostIp: "127.0.0.1"
     hostPort: 9300
```

Comment:
The curl command says to use port 9200, but hostPort is 9300. Intentional?

Also, of less concern to me:

1. `hostIp` is a hostname or an IP, so `host` is probably a better name for this field.
2. The comment `Specify the right clustername` is quite removed from the `clusterName`  field. I suggest ordering the comments/fields like this:

```
// For Aristo Solvers, ...
hostIp: ...
hostPort: ...

// Specify the right clustername. ...
clusterName: "..."
```